internal/server/models/organization.go

```diff
@@ -11,16 +14,31 @@ type Organization struct {
 	Model
 
 	Name      string `gorm:"uniqueIndex:idx_organizations_name,where:deleted_at is NULL"`
+	Domain    string `gorm:"uniqueIndex:idx_organizations_domain,where:deleted_at is NULL"`
 	CreatedBy uid.ID
 }
 
 func (o *Organization) ToAPI() *api.Organization {
 	return &api.Organization{
-		ID:      o.ID,
-		Created: api.Time(o.CreatedAt),
-		Updated: api.Time(o.UpdatedAt),
-		Name:    o.Name,
+		ID:     o.ID,
+		Name:   o.Name,
+		Domain: o.Domain,
+	}
+}
+
+var domainNameReplacer = regexp.MustCompile(`[^\da-zA-Z-]`)
+
+// TODO: is the model the right place for this? Should we do this in CreateOrg ?
+func (o *Organization) SetDefaultDomain() {
```

Comment:
We don't have to do this now, but I think this would be better in `data.CreateOrganization`. The two callers to this call create immediately after, and generally the create function in the `data` package is where we set defaults.